tests/acceptance/features/apiOcm/share.feature

```diff
@@ -618,3 +618,24 @@ Feature: an user shares resources using ScienceMesh application
         }
       }
       """
+
+  @issue-9898
+  Scenario: user lists permissions of a resource shared to a federated user
+    Given using server "LOCAL"
+    And "Alice" has created the federation share invitation
+    And using server "REMOTE"
+    And "Brian" has accepted invitation
+    And using server "LOCAL"
+    And user "Alice" has uploaded file with content "ocm test" to "/textfile.txt"
+    And user "Alice" has sent the following resource share invitation to federated user:
+      | resource        | textfile.txt                  |
+      | space           | Personal                      |
+      | sharee          | Brian                         |
+      | shareType       | user                          |
+      | permissionsRole | File Editor                   |
+      | federatedServer | @federation-ocis-server:10200 |
+    And using server "REMOTE"
+    And for user "Brian" the content of the file "/textfile.txt" of the space "Shares" should be "ocm test"
```

Comment:
Please, refer to https://github.com/owncloud/ocis/issues/10536
and you will need to do something like https://github.com/owncloud/ocis/blob/3ec2a4e695327ed444d4280d30c761b11418c7a2/tests/acceptance/bootstrap/SpacesTUSContext.php#L153-L156 done (WIP) in https://github.com/owncloud/ocis/pull/10415